deployment/arm/deploy-region.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
+set -eu
+
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
+if [[ $# -ne 4 ]]; then
```

Comment:
I believe this should be -ge - greater than or equal

Reply:
this is correct - my bad!

Reply:
Discussed offline with Jafar and figured this is not needed.